client/tailwind.config.js

```diff
@@ -5,7 +5,43 @@ export default {
     "./src/**/*.{js,ts,jsx,tsx}",
   ],
   theme: {
-    extend: {},
+    extend: {
+      typography: {
+        DEFAULT: {
+          css: {
+            blockquote: {
+              borderLeftColor: '#5D4A38',
+              borderLeftWidth: '4px',
+              backgroundColor: '#F7F5F0',
+              borderRadius: '0 0.375rem 0.375rem 0',
+              padding: '1rem 1.5rem 1rem 1.5rem',
+              margin: '1.5rem 0',
+              color: '#3A2E22',
+              fontStyle: 'normal',
+              position: 'relative',
+              quotes: '"\\201C""\\201D""\\2018""\\2019"',
+              '&::before': {
+                content: 'open-quote',
```

Comment:
it seem that this content will display the unnecessary big open quote after the vertical line of markdown as in the below screen shot. NG

![Screenshot 2025-05-19 at 11 25 47](https://github.com/user-attachments/assets/80d0603d-60de-48ba-b22c-2db00e05e655)